pom.xml

```diff
@@ -182,16 +151,16 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.6.0</version>
+                <version>3.10.1</version>
                 <configuration>
-                    <source>8</source>
```

Comment:
We still need to be able to run on java 8, but also on java 17. Target must stay at 8